src/site/content/en/blog/embed-best-practices/index.md

```diff
@@ -0,0 +1,314 @@
+
+```html
+<iframe src="https://example.com"
+       loading="lazy"
+       width="600"
+       height="400">
+</iframe>
+```
+
+The loading attribute supports the following values:
+
+* `lazy`: Indicates that the browser should defer loading the iframe. The browser will load the iframe when it is nearing the viewport. Use if the iframe is a good candidate for lazy-loading.
+* `eager`: Loads the iframe immediately. Use if the iframe is not a good candidate for lazy-loading. If the `loading` attribute has not been specified, this is the default behavior - except in [Lite mode](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid).
+* `auto`: The browser determines whether to lazy-load this frame.
+
+Browsers that don’t support the `loading` attribute ignore it, so you can apply native lazy-loading as a progressive enhancement. Browsers that support the attribute may have different implementations for the [distance-from-viewport](https://web.dev/browser-level-image-lazy-loading/#distance-from-viewport-thresholds) threshold (the distance at which the iframe starts loading).
+
+Following are some ways in which you can lazy load iframes for different types of embeds.
+
+* YouTube videos: To lazy-load a YouTube video player iframe,  include the `loading` attribute to the embed code provided by YouTube. Lazy loading the YouTube embed can save approximately 500 KB on the initial page load.
+
+```html
+<iframe src="https://www.youtube.com/embed/aKydtOXW8mI"
+   width="560" height="315"
+   loading="lazy"
+   title="YouTube video player"
+   frameborder="0"
+   allow="accelerometer; autoplay; clipboard-write; 
+            encrypted-media; gyroscope; picture-in-picture"
+   allowfullscreen>
+</iframe>
+```
+
+* Google Maps: To lazy-load a Google Map iframe, include the `loading` attribute in the code for the iframe embed generated by the [Google Maps Embed API](https://developers.google.com/maps/documentation/embed/get-started). Following is an example of the code with a placeholder for the Google Cloud API key.
+
+```html
+<iframe src="https://www.google.com/maps/embed/v1/place?key=API_KEY&q=PLACE_ID"
+   width="600" height="450"
+   style="border:0;"
+   allowfullscreen=""
+   loading="lazy">
+</iframe>
+```
+#### lazysizes library
+
+Because browsers use an embed’s distance-from-viewport, in addition to signals like [effective connection type](https://googlechrome.github.io/samples/network-information/) and Lite-mode, to decide when an iframe should be loaded, native lazy-loading can be inconsistent. If you need better control on the distance thresholds or you want to provide a consistent lazy-loading experience across browsers, you can use the [lazysizes](https://github.com/aFarkas/lazysizes) library. 
+
+_[lazysizes](https://github.com/aFarkas/lazysizes)_ is a fast, SEO-friendly lazy loader for both images and iframes. Once you have downloaded the component, it can be used with an iframe for a YouTube embed as follows.
```

Comment:
```suggestion
[lazysizes](https://github.com/aFarkas/lazysizes) is a fast, SEO-friendly lazy loader for both images and iframes. Once you have downloaded the component, it can be used with an iframe for a YouTube embed as follows.
```